pom.xml

```diff
@@ -1322,8 +1322,8 @@
 			 For instance, our version of Lucene is tied to the version of Hibernate Search. Similarly, our
 			 version of Hibernate Search must be compatible with the version of Hibernate and Hibernate must
 			 be compatible with our version of Spring. -->
-		<springVersion>6.2.5</springVersion>
-		<hibernateVersion>6.4.10.Final</hibernateVersion>
+		<springVersion>6.2.9</springVersion>
+		<hibernateVersion>6.6.22.Final</hibernateVersion>
```

Comment:
Could there be a reason why you did not upgrade this to `6.6.23.Final`?

Reply:
I must have missed that version, will update to that later today